CODEOWNERS

```diff
@@ -58,6 +58,8 @@
 
 /Workbooks/Individual*Storage/** @microsoft/iaasexp
 
+/Workbooks/LogAnalytics*Workspace/** @microsoft/loganalytics
```

Comment:
Since your team is not fully onboarded, it would be good to add us as a reviewer as well. This second group just means that when anyone from the workbooks team approves the PR, you can merge the changes in

`/Workbooks/LogAnalytics*Workspace/** @microsoft/loganalytics @microsoft/applicationinsights-devtools`

Reply:
i've added @microsoft/loganalytics with write access to the repo